### PR TITLE
ASR BLEU tool copied from ust branch into main

### PR DESCRIPTION
**Pull request by uralik**
## What does this PR do?

The ASR-BLEU tool is copied to main branch from ust branch. This is done upon request towards frequent uses (cc @xutaima).

### CODE REVIEW

File: examples/speech_to_speech/asr_bleu/asr_model_cfgs.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+            "ckpt_path": "https://dl.fbaipublicfiles.com/fairseq/wav2vec/wav2vec_vox_960h_pl.pt",
+            "dict_path": "https://dl.fbaipublicfiles.com/fairseq/wav2vec/dict.ltr.txt",
+            "model_type": "fairseq",
+            "lang": "en",
+            "post_process": "collapse"
+        }
+    },
+    "hok": {
+        "oct22": {
+            "desc": "Hokkien ASR model, for details check [TODO add paper link]",
```
Comment (xutaima) [inline examples/speech_to_speech/asr_bleu/asr_model_cfgs.json:14]: Is there a link? Maybe this [one](https://research.facebook.com/publications/hokkien-direct-speech-to-speech-translation/)?

**Review by xutaima — APPROVED** [review]

File: examples/speech_to_speech/asr_bleu/asr_model_cfgs.json (hunk 1 above)
Comment (xutaima) [reply examples/speech_to_speech/asr_bleu/asr_model_cfgs.json:14]: Thank you so much @uralik ! LGTM!

**Comment by xutaima** [comment]
Thank you so much @uralik ! LGTM!

File: examples/speech_to_speech/asr_bleu/asr_model_cfgs.json (hunk 1 above)
Comment (uralik) [reply examples/speech_to_speech/asr_bleu/asr_model_cfgs.json:14]: ah yes, now link exists, will add that!

**Comment by sravyapopuri388** [comment]
LGTM. The test failures don't look relevant to the PR. So approving!

**Review by sravyapopuri388 — APPROVED** [review]

**Pull request merged**